util/cache-nasa

```diff
@@ -17,15 +18,16 @@ fi
 
 set -euo pipefail
 
-echo "
-    CREATE DATABASE rai;
-" | psql -U postgres
+# echo "
+#     CREATE DATABASE rai;
+# " | psql -U postgres -h db
 
+#https://gis.stackexchange.com/a/341598
```

Comment:
Can we uncomment this script? Otherwise this will fail outside the docker context

Reply:
yes, we could remove the link, for Postgres version less than 12 , `postgis_raster` extension is included in the PostGIS extension.